File: modul1/soal10/PRAK110-2610817310007-Rahmat_Setiadi.c
#include <stdio.h>
#include <math.h>

int main()
{
    int base = 5;
    int height = 12;

    float b_side = sqrt(pow(base, 2) + pow(height, 2));
    float perimeter = base + height + b_side;
    float area = (base * height) / 2.0;

    printf("Diketahui :\n");
    printf("Alas = %d cm\n", base);
    printf("Tinggi = %d cm\n", height);

    printf("\nJawab :\n");
    printf("Sisi A = %d cm\n", height);
    printf("Sisi B = %.0f cm\n", b_side);
    printf("Sisi C = %d cm\n", base);
    printf("Keliling = %.0f cm\n", perimeter);
    printf("Luas = %.0f cm\n", area);
    return 0;
}

File: modul1/soal10/PRAK110-2610817310007-Rahmat_Setiadi.py
base = 5
height = 12

b_side = int((base**2 + height**2) ** 0.5)
perimeter = base + height + b_side
area = 0.5 * base * height

print("Diketahui :")
print("Alas =", base, "cm")
print("Tinggi =", height, "cm")

print("\nJawab :")
print(f"Sisi A = {height} cm")
print(f"Sisi B = {b_side} cm")
print(f"Sisi C = {base} cm")
print(f"Keliling = {perimeter} cm")
print(f"Luas = {area} cm²")
